### PR TITLE
Add Semaphore CI to the continuous integration documentation page

### PR DESCRIPTION
**Pull request by daveverwer**
Hello! I've been doing some work with [Semaphore](https://semaphoreci.com) recently and noticed that they are not listed on the continuous integration page so I popped together some documentation and here it is!

Thanks!
Dave

### CODE REVIEW

**Comment by daveverwer** [comment]
Regarding the failures:

1. I just signed a CLA 👍
2. Looking at the deploy log, I don't think this is caused by this PR but by the build system itself?

Let me know if you need me to do anything else. Thanks!

**Comment by daveverwer** [comment]
@googlebot I signed it!

File: docs/best-practices/continuous-integration/semaphore.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Semaphore CI Integration
+
+[Semaphore](https://semaphoreci.com) provides simple, fast continuous integration and delivery for your iOS apps. All [macOS machine images](https://docs.semaphoreci.com/article/162-macos-mojave-xcode-11-image) have fastlane tools pre-installed and getting up and running is quick and simple.
```
Comment (janpio) [inline docs/best-practices/continuous-integration/semaphore.md]:
```suggestion
[Semaphore](https://semaphoreci.com) provides simple, fast continuous integration and delivery for your iOS apps. All [macOS machine images](https://docs.semaphoreci.com/article/162-macos-mojave-xcode-11-image) have _fastlane_ pre-installed and getting up and running is quick and simple.
```

**Comment by daveverwer** [comment]
Accepted the nitpick. 👍 Thanks for the review!

**Comment by janpio** [comment]
Then let's see if Netlify's test deploy is indeed right or if this will build just fine in production...

**Comment by janpio** [comment]
Oh, it is indeed right:
https://circleci.com/gh/fastlane/docs/2225?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Ahem.

Be right back, fixing this ;)

**Comment by janpio** [comment]
I file like an idiot now, 3 weeks ago I already reported this: https://github.com/fastlane/docs/issues/884

**Comment by joshdholtz** [comment]
My apologies! This slipped through my backlog 😞 Will have this fixed soon 👌

**Comment by daveverwer** [comment]
Thanks for merging!

**Comment by daveverwer** [comment]
Actually, can I just clarify something please? I see that the [continuous integration](https://docs.fastlane.tools/best-practices/continuous-integration/#integrations) page now has the change from this PR deployed, but the link to the [Semaphore page](https://docs.fastlane.tools/best-practices/continuous-integration/semaphore/) is a 404.

Is that because of #884 or was there a problem with my link? It seems strange that half the changes in the PR were deployed.

**Comment by joshdholtz** [comment]
@daveverwer This is due to our CI failing to deploy on merge to master. I'm testing out fixes to our `Dockerfile` to get this back up and running right now 🤞

**Comment by joshdholtz** [comment]
@daveverwer Got a PR to fix this ☝️

**Comment by daveverwer** [comment]
Thanks, it all looks good now! 👍

**Pull request merged**